OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -50,6 +51,9 @@ public class SpawnActorOnDeathInfo : ConditionalTraitInfo
 			"lead to unexpected behaviour.")]
 		public readonly CVec Offset = CVec.Zero;
 
+		[Desc("Should an actor spawn after the player has been defeated (e.g. after surrendering)?")]
+		public readonly bool SpawnAfterDefeat = false;
```

Comment:
Tanks not spawning husks when a player surrenders is a bug.  This should default to true!